src/content/en/2020/markup.md

```diff
@@ -373,19 +400,19 @@ In our mobile data set of 6.3 million pages, around 0.9 million pages (14.01%) c
 <figcaption>{{ figure_link(caption="Obsolete elements with more than 10,000 uses.", sheets_gid="1972617631", sql_file="pages_element_count_by_device_and_obsolete_elements.sql") }}</figcaption>
 </figure>
 
-Why are these still alive—even `spacer` is still being used 1,584 times, and present on every 5,000th page. We know that Google has been using a `center` element on [their homepage](https://www.google.com/) [for 22 years](https://web.archive.org/web/19981202230410/https://www.google.com/) now, but does that find so many imitators?
+Why are these still alive? Even `spacer` is still being used 1,584 times, and present on every 5,000th page. We know that Google has been using a `center` element on [their homepage](https://www.google.com/) [for 22 years](https://web.archive.org/web/19981202230410/https://www.google.com/) now, but why are there so many imitators?
 
-<div class="note">
-  <h4><code>isindex</code></h4>
-  <p>If you were wondering: The <a href="https://www.w3.org/TR/html401/interact/forms.html#edef-ISINDEX"><code>isindex</code> element</a> was present once. (It was part of the HTML specs <a href="https://meiert.com/en/indices/html-elements/">until version 4.01, and of XHTML in 1.0</a>, yet only properly <a href="https://lists.w3.org/Archives/Public/public-whatwg-archive/2006Feb/0111.html">specified in 2006</a>, and <a href="https://github.com/whatwg/html/pull/1095">removed in 2016</a>.)</p>
-</div>
+#### `isindex`
+
+{# TODO(authors): What is the significance of this section? There are no new stats provided. #}
```

Comment:
"was present once", i.e. the new stat is 1 page used `isindex`. I'll leave it to the authors to decide the noteworthyness of this; seems like we could clarify the wording if we are to keep this section.

Reply:
Gotcha, I read that as "there was a time that `isindex` existed" not that "`isindex` is used on exactly one page". Agree it's worth clarifying.